_posts/2025-08-02-ordenar-score-asc-desc.md

```
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Ordena el score de Gestión comercial por valor.
+subtitle: Identifica de una manera más eficiente lo que buscas.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [Cliente, Ejecutivo, Factura, Gestión comercial, Score]
+comments: false
+author: Catalina Pinilla
+---
+
+## Gestión comercial - Score.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)